kustomize/components/alloydb/README.md

```diff
@@ -0,0 +1,152 @@
+# Integrate Online Boutique with AlloyDB
+
+By default the `cartservice` stores its data in an in-cluster Redis database. 
+Using a fully managed database service outside your GKE cluster (such as [AlloyDB](https://cloud.google.com/alloydb)) could bring more resiliency and more security.
+
+Note that because of AlloyDB's current connectivity, you'll need to run all this from a VM with
+VPC access to the network you want to use for everything (out of the box this should just use the
+default network). The Cloud Shell doesn't work because of transitive VPC peering not working.
+
+## Provision an AlloyDB database and the supporting infrastructure
+
+# Environmental variables needed for setup. These should be set in a .bashrc or similar as some of the variables are used in the application itself. Default values are supplied in this readme, but any of them can be changed.
+# PROJECT_ID should be set to the project ID that was created to hold the demo
```

Comment:
I think this should be wrapped in a code block.  Otherwise, your comments are interpreted as headings.

Reply:
Yup, changed it so the environment vars block are contained in the bash wrapper.